src/components/graph/graph.tsx

```diff
@@ -100,6 +101,7 @@ export class Graph {
     const fill = colorStops ? `url(#linear-gradient-${id})` : undefined;
     return (
       <svg
+        aria-hidden="true"
```

Comment:
given how all these SVG's share many attributes and classes this seems like a good opportunity for a utility function or functional component to normalize calcite svgs.